### ISSUE TITLE
v2.1.1 ERROR: Could not request to turn screen off if control is disabled

### ISSUE THREAD

**Issue by wicope** [Genymobile/scrcpy#4175]
v2.0 work
```
scrcpy \
--serial 1407bef7 \
--v4l2-sink=/dev/video0 --no-display \
--video-codec=h265 --video-encoder='OMX.qcom.video.encoder.hevc.cq' \
--lock-video-orientation=1 \
--video-bit-rate=60M \
--turn-screen-off \
--stay-awake \
--power-off-on-close \
--no-audio
```

v2.1.1 not work
```
scrcpy \
--serial 1407bef7 \
--v4l2-sink=/dev/video0 --no-display \
--video-codec=h265 --video-encoder='OMX.qcom.video.encoder.hevc.cq' \
--lock-video-orientation=1 \
--video-bit-rate=60M \
--turn-screen-off \
--stay-awake \
--power-off-on-close \
--no-audio
scrcpy 2.1.1 <https://github.com/Genymobile/scrcpy>
WARN: --no-display is deprecated, use --no-playback instead.
ERROR: Could not request to turn screen off if control is disabled

scrcpy --serial 1407bef7 --v4l2-sink=/dev/video0 --no-playback --video-codec=h265 --video-encoder='OMX.qcom.video.encoder.hevc.cq' --lock-video-orientation=1 --video-bit-rate=60M --turn-screen-off --stay-awake --power-off-on-close --no-audio
scrcpy 2.1.1 <https://github.com/Genymobile/scrcpy>
ERROR: Could not request to turn screen off if control is disabled
```

**Comment by rom1v** [comment]
Probably introduced by: https://github.com/Genymobile/scrcpy/commit/92483fe11b6fd6bae5ef775ccaff78fefa92aad4

It makes sense to disable controls when there is no display (because there is no way to control).. except for the initial `--turn-screen-off`. So the commit should probably be reverted.

**Comment by rom1v** [comment]
Fixed by 110b3a16f6d02124a4567d2ab79fcb74d78f949f. Please test the current `dev` branch.
